devops/docker/debian10-amd64/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM debian:buster
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt -y update && apt-get -y install software-properties-common
```

Comment:
nit: Any reason why *apt* over *apt-get* is used? Generally speaking, *apt* is intended for human end-users. As long as the Dockerfile only does the above, it is fine. Otherwise, it may be a good idea to use *apt-get* everywhere instead.

Reply:
I somehow thought i already made those changes to use apt over _apt-get/apt-cache/apt-*_. Although Simon seems to be suggesting the opposite (apt-* over apt). I wasn't aware of any functional differences between apt and apt-* but i definitely agree that we should stick to either of them and not mix-and-match as this can lead to issues when building the images.

Reply:
Indeed, for the above it will not have any major differences. Typically, *apt* adds things such as progress bars can affect the logs (rarely). I also found that it removes noise in the logs that is not useful for CI (but nice for humans). Overall, this is nitpicking, most important is that we standardize on one 😄.